operator/src/main/java/oracle/kubernetes/weblogic/domain/model/ClusterSpec.java

```diff
@@ -61,8 +61,8 @@ public class ClusterSpec extends BaseConfiguration implements Comparable<Cluster
       + "domain-lifecycle/startup/#starting-and-stopping-servers.")
   private ServerStartPolicy serverStartPolicy;
 
-  @Description(
-      "The maximum number of cluster members that can be temporarily unavailable. Defaults to 1.")
+  @Description("The maximum number of cluster members that can be temporarily unavailable. "
+      + "Defaults to `domain.spec.maxClusterUnavailable`, which defaults to 1.")
   @Range(minimum = 1)
```

Comment:
Should we also add '@Default(intDefault = 1)', same as we did for the DomainSpec?

Reply:
this is the cluster level setting, which now defaults to the domain level setting, which in turn defaults to 1.